sites/tremend.py

```diff
@@ -1,46 +1,48 @@
 import requests
-import json
 from bs4 import BeautifulSoup
 from utils import *
 from getCounty import *
 
 url = 'https://tremend.com/careers/'
 company = 'tremend'
 
-final_jobs = []
-
 r = requests.get(url)
 soup = BeautifulSoup(r.content, 'html.parser')
 
-job_elements = soup.find('div', id='jobs')
+final_jobs = []
+job_elements = soup.find('div', id='jobs').find_all('div', class_='career-wrapper')
 
-for job in job_elements.find_all('div', class_='career-wrapper'):
-    job_title = job.find('h3').text
-    job_link = url + job.find('a')['href']
+for job in job_elements:
+    job_title = job.find('h3').text.strip()
+    job_link = 'https://tremend.com/careers/' + job.find('a')['href'].strip('/')
     location_text = job.find('p', id='location-word').text.strip().replace('Location', '').strip()
-    remote = location_text == 'Remote'
+
+    remote = 'Remote' in location_text
```

Comment:
campus remote trebuie sa conțină strungul "Remote" nu valoarea bool true.
<img width="1440" alt="Screenshot 2023-11-16 at 15 39 38" src="https://github.com/peviitor-ro/based_scraper_py/assets/67306273/90ff13ed-abf0-46a0-a79f-3ac34a59a3c9">